applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1300,20 +1228,69 @@ int le_audio_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_
 	return -ENOTSUP;
 }
 
-int le_audio_play_pause(void)
+int le_audio_play(void)
 {
-	int ret;
+	int ret_left = 0;
+	int ret_right = 0;
 
-	if (IS_ENABLED(CONFIG_STREAM_BIDIRECTIONAL)) {
-		LOG_WRN("Play/pause not supported for bidirectional mode");
-	} else {
-		ret = ble_mcs_play_pause(NULL);
-		if (ret) {
-			LOG_WRN("Failed to change streaming state");
-			return ret;
+	if (ep_state_check(headsets[AUDIO_CH_L].sink_stream.ep, BT_BAP_EP_STATE_QOS_CONFIGURED)) {
+		ret_left = bt_bap_stream_enable(&headsets[AUDIO_CH_L].sink_stream,
+						lc3_preset_sink.codec.meta,
+						lc3_preset_sink.codec.meta_count);
+
+		if (ret_left) {
+			LOG_WRN("Failed to enable left stream: %d", ret_left);
+		}
+	}
+
+	if (ep_state_check(headsets[AUDIO_CH_R].sink_stream.ep, BT_BAP_EP_STATE_QOS_CONFIGURED)) {
+		ret_right = bt_bap_stream_enable(&headsets[AUDIO_CH_R].sink_stream,
+						 lc3_preset_sink.codec.meta,
+						 lc3_preset_sink.codec.meta_count);
+
+		if (ret_right) {
+			LOG_WRN("Failed to enable right stream: %d", ret_right);
+		}
+	}
+
+	playing_state = true;
+
+	if (ret_left || ret_right) {
```

Comment:
If both failed, then `playing_state` should probably not be set to `true`

Reply:
Will do: 
```suggestion
	if (ret_left || ret_right) {
	    return -EIO;
	} else if (!ret_left && !ret_right) {
	    playing_state = true;
	}
```